src/typesetting/formats.ts

```diff
@@ -31,7 +31,7 @@ export const link = format({
   name: 'link',
   selector: 'a[href]',
   greedy: false,
-  commands: editor => (link: string) => editor.toggleTextFormat({ link }),
+  commands: editor => (link: string) => editor.toggleTextFormat({ link: typeof link === 'string' ? link : true }),
```

Comment:
Is this so that a link can be toggled off without passing in text? Perhaps we should fix this from a toggle to a set.
```ts
(link: string) => editor.setTextFormat({ link: link || null })
```

This way you can use `commands.link('example.com');` to set a link to another thing. The old way would toggle it off which is not what we want. I think this was a bug.

Reply:
I think you're right.

Reply:
But I think you mean "formatText"?

Reply:
Actually it needs to be toggle, just with the right checks.